pkg/netflow/flowaggregator/flowstore.go

```diff
@@ -0,0 +1,56 @@
+package flowaggregator
+
+import (
+	"sync"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	"github.com/DataDog/datadog-agent/pkg/netflow/common"
+)
+
+type flowStore struct {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏This class looks more like an _accumulator_ than a _store_... Would `flowAccumulator` be a more meaningful name?‏

Reply:
+1, fixed here https://github.com/DataDog/datadog-agent/pull/10976/commits/21d0e7d8061c33a5f32d6a31b36461e4af99eafc